html/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 
 <head>
-  <title>CNF Certification Test Parser</title>
```

Comment:
Why not use "Red Hat Best Practices Test Suite for Kubernetes" instead of certsuite everywhere?

Reply:
i prefer this 
Red Hat Best Practices Certification without Test Suite